LICENSE

```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2025 foyer Project Authors
```

Comment:
https://github.com/foyer-rs/foyer/pull/913/files#diff-c693279643b8cd5d248172d9c22cb7cf4ed163a3c98c8a3f69c2717edd3eacb7L179

Do we need to update the appendix part of the original license text? 👀

Reply:
I'm stupid

Reply:
Don't worry, I've done the same thing. Actually, I'm still not sure about it. QwQ